text/0024-cron.md

```diff
@@ -0,0 +1,196 @@
+- Feature Name: cron
+- Start Date: 2019-04-12
+- RFC PR: <!-- (leave this empty) -->
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Cron generates new dataset verions with a damonized service that schedules & executes updates. Using cron, qri can keep data up to date on behalf of a user, and provide notifications of significant changes to datasets.
+
+The goals of this RFC are twofold:
+* define `qri cron`, a service for automating dataset updates
+* define the characteristics & requirements of running a "fog service"
```

Comment:
"fog service" is a good abstraction of what's happening, but I wonder if the word "fog" has too many negative associations. I like that it's akin to "cloud" but literally closer to earth. But it can be interpreted as "confusing" or "shady" or "opaque"

Reply:
Personally, I'm ok with the "confusing" side of things. I'm hoping we can stay lighthearted about the whole thing. To me the "abstraction phrase" needs to emphasize honesty over wanting to be placed in a positive light